cpu/native/Kconfig

```diff
@@ -89,3 +89,4 @@ rsource "backtrace/Kconfig"
 endmenu # Native modules
 
 rsource "periph/Kconfig"
+rsource "mtd/Kconfig"
```

Comment:
Can we have the `source`s in alphabetical order? :)

Reply:
Directly amended it.